Formula/hashcat.rb

```diff
@@ -0,0 +1,21 @@
+class Hashcat < Formula
+  desc "World's fastest and most advanced password recovery utility"
+  homepage "https://hashcat.net/hashcat/"
+  url "https://hashcat.net/files/hashcat-#{version}.tar.gz"
+  version "3.10"
```

Comment:
We prefer to omit the `version` line and let it be parsed from the `url`. An explicit `version` is typically only for weird filenames where the version can't be parsed automatically.